websites/M/MangaHere/presence.ts

```diff
@@ -0,0 +1,104 @@
+const presence = new Presence({
+  clientId: "831262912815300638"
+}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const data: PresenceData = {
+    largeImageKey: "mangahere",
+    startTimestamp: browsingStamp
+  },
+    pathname = document.location.pathname,
+    hostname = document.location.hostname,
+    ganres = ["martial-arts", "action", "school-life",
+              "sci-fi", "yoi", "shotacon", "mystery", "shoujo",
+              "ecchi", "doujinshi", "lolicon", "adventure", "romance",
+              "gender-bender", "harem", "sports", "webtoons", "comedy",
+              "shounen-ai", "josei", "shoujo-ai", "adult", "fantasy", "supernatural",
+              "psychological", "yuri", "one-shot", "historical", "drama",
+              "seinen", "mature", "smut", "horror", "shounen", "slice-of-life",
+              "tragedy", "mecha"
+            ];
+
+  if (hostname === "www.mangahere.cc") {
```

Comment:
I just noticed that you don't need this `if` statement since you have only one domain specified in metadata.

Reply:
Oh, exactly, i didn't delete it since I had 2 domains at first. On it :)